specs/windows/drivers.table

```diff
@@ -0,0 +1,15 @@
+table_name("drivers")
+description("Lists all installed and loaded Windows Drivers and its relevant data")
+schema([
+    Column("name", TEXT, "Driver name, drivers not managed by SCM are displayed as <driver_name>"),
+    Column("display_name", TEXT, "Driver Display name"),
+    Column("status", TEXT, "Driver Current status: STOPPED, START_PENDING, STOP_PENDING, RUNNING, CONTINUE_PENDING, PAUSE_PENDING, PAUSED"),
+    Column("start_type", TEXT, "Driver start type: BOOT_START, SYSTEM_START, AUTO_START, DEMAND_START, DISABLED"),
+    Column("path", TEXT, "Path to Driver module"),
+    Column("driver_type", TEXT, "Driver Type: KERNEL, FILE_SYSTEM"),
+    Column("inf", TEXT, "INF File: Associated inf file"),
```

Comment:
Oh perfect, I missed the existing `path` column above, thanks for correcting me!